src/oauth2c.erl

```diff
@@ -89,6 +93,13 @@
 
 %%% API ========================================================================
 
+start() ->
+    oauth2c_token_cache_gen_server:start().
+start(State) ->
+    oauth2c_token_cache_gen_server:start(State).
+stop() ->
```

Comment:
Blank lines between functions are nice. Except for some special cases.

Reply:
Also, there is no need to implement a `stop` function if you have a supervision tree. OTP handles the stopping for you, and for this simple case you won't need to do anything on shutdown.

Reply:
I have added a supervisor and removed the stop function.